dataactcore/models/domainModels.py

```diff
@@ -27,6 +28,27 @@ class ObjectClass(Base):
     object_class_code = Column(Text,nullable=False,index=True,unique=True)
     object_class_name = Column(Text)
 
+class SF133(Base):
+    __tablename__ = "sf_133"
+    sf133_id = Column(Integer,primary_key=True)
+    agencyidentifier = Column(Text, nullable=False)
+    allocationtransferagencyidentifier = Column(Text)
+    availabilitytypecode = Column(Text)
+    beginningperiodofavailability = Column(Text)
+    endingperiodofavailability = Column(Text)
+    mainaccountcode = Column(Text, nullable=False)
+    subaccountcode = Column(Text, nullable=False)
+    tas = Column(Text, nullable=False, default=concatTas, onupdate=concatTas, index=True)
+    fiscal_year = Column(Text)
+    period = Column(Text)
+    line = Column(Integer,nullable=False)
+    amount = Column(Text,nullable=False,default=0,server_default="0")
```

Comment:
Changed back to Numeric, and removing commas and converting empty to 0 to fit data into Numeric.